code/modules/atmospherics/ZAS/zas_extras/inflatable.dm

```diff
@@ -291,14 +303,115 @@
 			src.transfer_fingerprints_to(R)
 			qdel(src)
 
+//inflatable shelters from vg
+/obj/structure/inflatable/shelter
+	name = "inflatable shelter"
+	density = TRUE
+	anchored = FALSE
+	opacity = 0
+	can_buckle = TRUE
+	icon_state = "shelter_base"
+	undeploy_path = /obj/item/inflatable/shelter
+	var/list/exiting = list()
+	var/datum/gas_mixture/cabin_air
+
+/obj/structure/inflatable/shelter/Initialize()
+	. = ..()
+	cabin_air = new
+	cabin_air.volume = CELL_VOLUME / 3
+	cabin_air.temperature = T20C + 20
```

Comment:
This results in an internal temperature of 40C, is this intentional?

Reply:
the temperature is copied from vg's original shelter, since it's meant to help warm people up fast if they were recently spaced. I'm not actually sure if this matters or helps in tg medical, but it can't hurt